src/libcore/ops.rs

```diff
@@ -235,6 +235,44 @@ pub trait Drop {
 /// }
 /// ```
 ///
+/// Here is an example of the same `Point` struct implementing the `Add` trait
+/// using generics.
+///
+/// ```
+/// use std::ops::Add;
+///
+/// #[derive(Debug)]
```

Comment:
The other examples derived Debug, so I followed the same path. Do you want me to remove Debug in this example and the other examples in std::Ops?